src/main/scala/pirate/Runners.scala

```diff
@@ -16,10 +16,13 @@ trait Runners {
         }
         case ParseErrorShowHelpText(s) => IO {
           s match {
-            case None      => Console.err.print(Usage.print(command, ctx))
-            case Some(sub) => Console.err.print(Usage.print(command, sub :: ctx))
+            case None      => Console.out.print(Usage.print(command, ctx))
+            case Some(sub) => Console.out.print(Usage.print(command, sub :: ctx))
```

Comment:
Are there any error cases where this is triggered, or is it only the "I successfully requested help" case. If the later I agree.